mortgage/mortgage_ibis.py

```diff
@@ -199,45 +200,71 @@ def etl_ibis(
-        elif import_mode == "fsi":
-            year, quarter = MortgagePandasBenchmark.split_year_quarter(dfiles_num)
-            omnisci_server_worker._conn.create_table_from_csv(
-                f"{table_prefix}_acq",
-                f"{mb.acq_data_path}/Acquisition_{year}Q{quarter}.txt",
-                acq_schema,
-                database_name,
-                delimiter=",",
-                header="true",
-                fragment_size=fragments_size[0],
-            )
-            # FIXME: handle cases when quarter perf is split in two files
-            omnisci_server_worker._conn.create_table_from_csv(
-                f"{table_prefix}_perf",
-                f"{mb.perf_data_path}/Performance_{year}Q{quarter}.txt",
-                perf_schema,
-                database_name,
-                delimiter=",",
-                header=True,
-                fragment_size=fragments_size[1],
-            )
-        etl_times["t_readcsv"] = timer() - t0
-        etl_times["t_connect"] += omnisci_server_worker.get_conn_creation_time()
-
-    # Second connection - this is ibis's ipc connection for DML
-    t0 = timer()
-    omnisci_server_worker.connect_to_server(database_name, ipc=ipc_connection)
-    acq_table = omnisci_server_worker.database(database_name).table(f"{table_prefix}_acq")
-    perf_table = omnisci_server_worker.database(database_name).table(f"{table_prefix}_perf")
-    etl_times["t_connect"] += timer() - t0
-
-    t_etl_start = timer()
-    ibis_df = run_ibis_workflow(acq_table, perf_table, leave_category_strings)
-    etl_times["t_etl"] = timer() - t_etl_start
-
-    return ibis_df, mb, etl_times
+    dfs = []
+    for data_file_num in range(dfiles_num):
+        year, quarter = MortgagePandasBenchmark.split_year_quarter(data_file_num)
+        # read acq file
+        if create_new_table:
+            t0 = timer()
+            if import_mode == "copy-from":
+                raise ValueError("COPY FROM does not work with Mortgage dataset")
+            elif import_mode == "pandas":
+                raise NotImplementedError("Loading mortgage for ibis by Pandas not implemented yet")
+            elif import_mode == "fsi":
+                omnisci_server_worker._conn.create_table_from_csv(
+                    f"{table_prefix}_acq",
+                    f"{mb.acq_data_path}/Acquisition_{year}Q{quarter}.txt",
+                    acq_schema,
+                    database_name,
+                    delimiter=",",
+                    header="true",
+                    fragment_size=fragments_size[0],
+                )
+            etl_times["t_readcsv"] += timer() - t0
+            etl_times["t_connect"] += omnisci_server_worker.get_conn_creation_time()
+        # iterate perf files
+        perf_files = [f for f in Path(mb.perf_data_path).iterdir()
+                      if f.match('Performance_%sQ%s.txt*' % (str(year), str(quarter)))]
+        for perf_file in perf_files:
+            print(str(perf_file))
```

Comment:
```suggestion
            print(f'Loading {perf_file}...')
```
to make it more obvious what's happening